testsuite/scripts/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+if(NOT DEFINED TEST_NP)
+  include(ProcessorCount)
+  ProcessorCount(NP)
+  math(EXPR TEST_NP "${NP}/2 + 1")
+endif()
+
+set(TEST_FILE_CONFIGURED_IMPORTLIB_WRAPPER ${CMAKE_CURRENT_BINARY_DIR}/test_importlib_wrapper.py)
+configure_file(importlib_wrapper.py ${CMAKE_CURRENT_BINARY_DIR}/importlib_wrapper.py)
+configure_file(test_importlib_wrapper.py ${TEST_FILE_CONFIGURED_IMPORTLIB_WRAPPER})
+
+macro(PYTHON_SCRIPTS_TEST)
```

Comment:
we should add a ```sample``` / ```tutorial``` prefix for these tests to distinguish them from the rest of the python tests

Reply:
like so?
```
Test project /work/jgrad/espresso/build/testsuite/scripts/samples
      Start  1: importlib_wrapper
 1/59 Test  #1: importlib_wrapper ............................   Passed    0.60 sec
      Start  2: sample_save_checkpoint
 2/59 Test  #2: sample_save_checkpoint .......................   Passed    4.35 sec
      Start  3: sample_store_properties
 3/59 Test  #3: sample_store_properties ......................   Passed   10.78 sec
```
`importlib_wrapper` is neither a tutorial nor a sample, and won't appear in `check_python`

Reply:
yes, looks good